merge_sort/C/merge_sort.c

```diff
@@ -0,0 +1,73 @@
+#include <stdio.h>  // for printf function
+
+// Function prototypes
+void mergesort(int[], int, int);
+void merge(int[], int, int, int);
+
+int main() {
+	int arr_size = 6;
+	int arr[6] = {10, 9, 8, 7, 6, 5};
+	mergesort(arr, 0, arr_size);
+
+    // Print sorted array
```

Comment:
Line contains following spacing inconsistencies:
- Spaces used instead of tabs.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp2ezo8ne3/merge_sort/C/merge_sort.c
+++ b/tmp/tmp2ezo8ne3/merge_sort/C/merge_sort.c
@@ -9,7 +9,7 @@
 	int arr[6] = {10, 9, 8, 7, 6, 5};
 	mergesort(arr, 0, arr_size);
 
-    // Print sorted array
+	// Print sorted array
 	for (int i = 0; i < arr_size; i++) {
 		printf("%d ", arr[i]);
 	}
```